contents/handbook/company/values.md

```diff
@@ -32,7 +32,7 @@ Our platform is built for developers, and we use GitHub to build a large communi
 
 Whether you're a designer or you're in operations, we will encourage and help you to be able to make basic changes to our website and docs on GitHub.
 
-## Tread on toes
+## Step on toes
```

Comment:
This breaks a link here: contents/handbook/people/feedback.md

Also might be useful to search for tread on toes and make the terminology consistent elsewhere.

I prefer "step" too.